custom-login/src/app/home/home.component.html

```diff
@@ -17,8 +17,8 @@ <h1 class="ui header">Custom Login Page with Sign In Widget</h1>
   <button id="login-button" class="ui primary button" role="button" routerLink="/login">Login</button>
 </div>
 
-<div *ngIf="oktaAuth.isAuthenticated()">
-  <p>Welcome back, {{idToken.claims.name}}!</p>
+<div *ngIf="isAuthenticated && userName">
```

Comment:
Are we checking for `userName` here because we don't want the page to update until the `async` call completes?

Reply:
Correct, it looks goofy otherwise